app/models/node.rb

```diff
@@ -750,6 +750,19 @@ def add_tag(tagname, user)
                                  uid: user.uid,
                                  date: DateTime.now.to_i,
                                  nid: id)
+
+          # Adding lat/lon values into node table
+          if tag.valid?
+            if tag.name.split(':')[0] == 'lat'
+              tagvalue = tag.name.split(':')[1]
+              self.update_attribute(:latitude, tagvalue)
+              self.update_attribute(:precision, decimals(tagvalue).to_s)
```

Comment:
Ah, but is precision stored as a string? Shouldn't it be an int? Perhaps we should change it to an int; sorry if I missed this in the migration. Thanks!